DigitalLearningSolutions.Data/Services/SupportTicketService.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.DataServices.UserDataService;
+
+    public interface ISupportTicketService
+    {
+        int GetNumberOfTicketsForCentreAdmin(int centreId, int adminId);
+    }
+
+    public class SupportTicketService : ISupportTicketService
+    {
+        private readonly ISupportTicketDataService ticketDataService;
+        private readonly IUserDataService userDataService;
+
+        public SupportTicketService(ISupportTicketDataService ticketDataService, IUserDataService userDataService)
+        {
+            this.ticketDataService = ticketDataService;
+            this.userDataService = userDataService;
+        }
+
+        public int GetNumberOfTicketsForCentreAdmin(int centreId, int adminId)
+        {
+            var admin = userDataService.GetAdminUserById(adminId);

```

Comment:
We should be able to determine if an admin is a centre manager or not from their custom claims. Do you think it is worth grabbing that from the custom claim instead? Would saving making a call to the DB. Think the existing extension method for it also checks if they are a SuperAdmin, so might need to double check if we want that.

Though I've also noticed we make a call to `GetAdminUserById` in the controller action anyway.